tests/integration/cmd_pref_config_test.go

```diff
@@ -359,10 +359,6 @@ var _ = Describe("odo preference and config command tests", func() {
 			output = helper.CmdShouldPass("odo", "preference", "unset", "buildtimeout", "-f")
 			Expect(output).ToNot(ContainSubstring(promtMessageSubString))
 		})
-		It("prompt should appear when non-preference command is run", func() {
-			output := helper.CmdShouldPass("odo", "create", "nodejs", "--context", commonVar.Context)
-			Expect(output).To(ContainSubstring(promtMessageSubString))
-		})
```

Comment:
Can we unset(in setup) and set(in teardown) the ODO_DISABLE_TELEMETRY env to test this scenario instead of removing it?

Reply:
Originally I deleted this to make sure that we don't send something out even by accident.
Now when we also have https://github.com/openshift/odo/pull/4585 I can put it back and try unsettling ODO_DISABLE_TELEMETRY

Reply:
There is also problem that now the message is now shown if odo is not executed in terminal

Reply:
In that case, I'd rather we comment out the test with a reason than remove it.